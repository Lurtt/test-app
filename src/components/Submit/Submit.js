import styled from 'styled-components'

export default styled.input.attrs({
  type: 'submit',
})`
  height: 36px;
  margin: 2rem 0;
  padding: 0 1.2rem;
  color: #fff;
  background-color: #26a69a;
  border-radius: 2px;
  text-transform: uppercase;
  text-align: center;
  letter-spacing: 0.5px;
  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
    0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
  border: none;
  outline: 0;
  cursor: pointer;
  transition: 0.3s ease-out;

  &:focus,
  &:hover {
    background-color: #2bbbad;
  }
`
