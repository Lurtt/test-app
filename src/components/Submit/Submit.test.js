import React from 'react'
import renderer from 'react-test-renderer'

import Submit from './'

describe('<Submit />', () => {
  test('Should render', () => {
    const result = renderer.create(<Submit value="Click me!" />).toJSON()
    expect(result).toMatchSnapshot()
  })
})
