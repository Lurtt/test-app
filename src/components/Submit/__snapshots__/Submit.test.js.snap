// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Submit /> Should render 1`] = `
<input
  className="sc-bdVaJa iyuqSV"
  type="submit"
  value="Click me!"
/>
`;
